contracts/src/Gateway.sol

```diff
@@ -53,7 +53,7 @@ contract Gateway is IGateway, IInitializable {
     address internal immutable AGENT_EXECUTOR;
```

Comment:
I'd suggest making`AGENT_EXECUTOR` also public, then in https://github.com/Snowfork/snowbridge/blob/fd2fd86746ec91c8ce4e6c345cf925d6dbbd07a7/contracts/src/DeployGatewayLogic.sol#L28 when to upgrade with a new Gateway logic, there is no need to create it again.

Reply:
Agree, lets make it public too 👍

Reply:
Addressed.